apis/go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/openstack-k8s-operators/mariadb-operator/api v0.5.0
 	github.com/openstack-k8s-operators/neutron-operator/api v0.5.1-0.20241112143215-294abc29ae85
 	github.com/openstack-k8s-operators/nova-operator/api v0.5.0
-	github.com/openstack-k8s-operators/octavia-operator/api v0.5.0
+	github.com/openstack-k8s-operators/octavia-operator/api v0.5.1-0.20241119144914-248c7edb4442
```

Comment:
shouldn't it be 0.5.0-something? how do we check that this version is the right one?

Reply:
I get from: 

```
froyo@fedora ~ $ go list -m -json github.com/openstack-k8s-operators/octavia-operator/api@18.0-fr1 | jq '.|"\(.Path) \(.Version)"'
"github.com/openstack-k8s-operators/octavia-operator/api v0.5.1-0.20241119144914-248c7edb4442"
```